en/intro-concepts.md

```diff
@@ -196,6 +196,17 @@ object in the fabric will be associated with by default (unless specified
 otherwise).
 
 
+## DHCP relay
+
+A DHCP relay, or relay agent, is a network device that forwards requests and
+replies between a DHCP client and a DHCP server when both are not on the same
+physical subnet.
+
+See [isc-dhcp-relay](http://packages.ubuntu.com/xenial/isc-dhcp-relay) and
+[dhcp-helper](http://packages.ubuntu.com/xenial/dhcp-helper) for well-known
```

Comment:
I think this is missing something. for - a - well-known software implementation?